src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -251,6 +251,137 @@ auto createForceModel( PMB, Fracture, ParticleType particles,
         delta, K, G0, temp, alpha, temp0 );
 }
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, DynamicTemperature, TemperatureType>
+    : public ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+{
+    using base_type =
+        ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>;
+    using base_model = PMB;
+    using fracture_type = Elastic;
+    using thermal_type = DynamicTemperature;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Thermal parameters
+    using base_type::alpha;
+    using base_type::temp0;
+    double thermal_coeff;
```

Comment:
This coefficient may need update for different models.

Reply:
Fixed